packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -1372,6 +1372,82 @@ describe('meeting', () => {
       });
     });
 
+    describe('joinMeeting', () => {
+      const chatInfoParams: meeting.ChatInfo = {
+        messageId: 'mockMessageId',
+        replyChainMessageId: 'mockReplyChainMessageId',
+        threadId: 'mockThreadId',
+      };
+
+      const mockJoinMeetingParams: meeting.JoinMeetingParams = {
+        joinWebUrl: new URL('https://example.com'),
+        subject: 'mockSubject',
+        source: meeting.EventActionSource.Other,
+        chatInfo: chatInfoParams,
+      };
+
+      it('should reject if joinMeetingParams is not provided', async () => {
+        const response = meeting.joinMeeting(null);
+        await expect(response).rejects.toEqual('Invalid joinMeetingParams');
+      });
+
+      it('should reject if chatInfo is not provided', async () => {
+        const response = meeting.joinMeeting({
+          ...mockJoinMeetingParams,
+          chatInfo: null,
+        });
+        await expect(response).rejects.toEqual('Invalid joinMeetingParams');
+      });
+
+      it('should reject if threadId is not provided', async () => {
+        const response = meeting.joinMeeting({
+          ...mockJoinMeetingParams,
+          chatInfo: {
+            ...chatInfoParams,
+            threadId: null,
+          },
+        });
+        await expect(response).rejects.toEqual('Invalid joinMeetingParams');
+      });
+
+      it('should reject if joinWebUrl is not provided', async () => {
+        const response = meeting.joinMeeting({
+          ...mockJoinMeetingParams,
+          joinWebUrl: null,
+        });
+        await expect(response).rejects.toEqual('Invalid joinMeetingParams');
+      });
+
+      it('should reject if joinWebUrl is empty', async () => {
+        const response = meeting.joinMeeting({
+          ...mockJoinMeetingParams,
+          chatInfo: {
+            ...chatInfoParams,
+            threadId: '',
+          },
+        });
+        await expect(response).rejects.toEqual('Invalid joinMeetingParams');
+      });
+
+      it('should reject if joinWebUrl is empty', async () => {
```

Comment:
You should have a unit test that verifies this is rejected if joinWebUrl has a value of 'this is not a valid url' since you have code that will throw an exception in that case.

Reply:
Added a UT for this.